osu.Game/Configuration/SessionStatics.cs

```diff
@@ -16,6 +16,7 @@ protected override void InitialiseDefaults()
         {
             SetDefault(Static.LoginOverlayDisplayed, false);
             SetDefault(Static.MutedAudioNotificationShownOnce, false);
+            SetDefault(Static.LowBatteryNotificationShownOnce, false);
```

Comment:
One thing to note is that if the app is never closed, this will not be reset, meaning potentially it will not be displaying when the user may want it to. I'm not sure if that's an issue which should be solved in this PR, or whether it's a `SessionStatics` issue (one could argue the other features using this class suffer from the same flaw, and should be reset when the app goes inactive or after a certain period of time).

Leaning towards the latter.